test/controllers/vendor_controller_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
These test files were auto-generated by ruby. The contents of this pr is just a set of static frontend pages relating to the vendors so no tests were added.